src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let total = 40 * days;
+
+  if (days >= 3 && days < 7) {
+    total -= 20;
+  } else if (days >= 7) {
+    total -= 50;
+  }
```

Comment:
in this case its okay, but multiple `else if`s are definitely not easy to read. 
what is 40? what is 20? what is 50? imagine somebody asks you to refactor the code. you don't have any idea what these numbers mean.

readability and understandable code are of the most importance.
then you can think about how you can shorten it, but only if it does not hinder the above